OpenRA.Mods.Common/Traits/Player/StrategicVictoryConditions.cs

```diff
@@ -23,7 +23,7 @@ public class StrategicPoint { }
 	public class StrategicVictoryConditionsInfo : ITraitInfo, Requires<MissionObjectivesInfo>
 	{
 		[Desc("Amount of time (in game ticks) that the player has to hold all the strategic points.", "Defaults to 5 minutes.")]
-		public readonly int TicksToHold = 25 * 60 * 5;
+		public readonly int HoldDuration = 25 * 60 * 5;
```

Comment:
Makes sense, I'll see about updating them shortly. Is a new commit necessary or should I just amend the refactor commit?